src/pages/Explorer/Components/DeleteConfirmationDialog.svelte

```diff
@@ -0,0 +1,46 @@
+<script context="module">
+  import { dialogs } from 'webkit/ui/Dialog'
+  import DeleteConfirmationDialog from './DeleteConfirmationDialog.svelte'
+
+  export function showDeleteConfirmationDialog(itemType, onDelete, onComplete) {
+    dialogs.show(DeleteConfirmationDialog, { itemType, onDelete, onComplete })
+  }
+</script>
+
+<script>
+  import Dialog from 'webkit/ui/Dialog'
+  import Svg from 'webkit/ui/Svg/svelte'
+
+  let closeDialog
+
+  $: ({ itemType, onDelete, onComplete } = $$props)
+</script>
+
+<Dialog {...$$props} noTitle bind:closeDialog>
+  <div class="dialog">
+    <div class="row justify v-center h4 txt-m mrg--b mrg-s c-black">
+      Do you want to delete this {itemType}?
+      <Svg id="close" w="12" on:click={closeDialog} class="btn" />
+    </div>
+    <div class="column hv-center body-2 c-waterloo mrg--b mrg-xl">
```

Comment:
1. `column` is redundant;
2. `[D-0-1]` Change the order of the margin classes: 1) `mrg-[size]` 2) `mrg-[side]`